docker/supabase/volumes/db/init/05-github-schema.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE IF NOT EXISTS github_users (
+  id serial PRIMARY KEY,
+  username text, -- the user name of the user, known as the `login` field in GitHub API responses
+  user_type text check ( user_type IN ('User', 'Organization') )
+);
+
+CREATE TABLE IF NOT EXISTS github_repositories (
+  id serial PRIMARY KEY,
+  repo_name text,
+  owner integer references github_users ON DELETE CASCADE
+);
+
+-- Although pull requests have different features from
+-- regular issues, they can be treated the same for our purposes
```

Comment:
From the [Issues API docs](https://docs.github.com/en/rest/reference/issues), so probably you're right to call these issues:

> Note: GitHub's REST API v3 considers every pull request an issue, but not every issue is a pull request. For this reason, "Issues" endpoints may return both issues and pull requests in the response. You can identify pull requests by the pull_request key. Be aware that the id of a pull request returned from "Issues" endpoints will be an issue id. To find out the pull request id, use the "[List pull requests](https://docs.github.com/rest/reference/pulls#list-pull-requests)" endpoint.

And from the [Pull API docs](https://docs.github.com/en/rest/reference/pulls):

> Every pull request is an issue, but not every issue is a pull request. For this reason, "shared" actions for both features, like manipulating assignees, labels and milestones, are provided within [the Issues API](https://docs.github.com/en/rest/reference/issues).

I think it might be good to expand your comment above to better explain this, as I don't know that everyone would understand this intuitively (calling it `github_issues` will confuse some devs).